### DISCUSSION TITLE
Support user provided minimum `ctid` for Postgres scanner

### DISCUSSION THREAD

**Discussion by ben-pr-p** [duckdb/duckdb#9486] [Ideas]
Thanks for your wonderful work on the Postgres scanner - just tried it out on some massive tables and it was more performant than anything I would have written myself.

For my use case, I'm continually bringing in the latest data from Postgres into a DuckDB database. I'm using the `postgres_scan_pushdown` function and provided a filter on an indexed `created_at`. This is working well, but DuckDB is still wasting some time scanning the bottom range of `ctid`s for the table since I know as a user that any record created before my provided `created_at` has a higher `ctid` than some `ctid` I fetched the last time I ran this code.

If I could do:
```
INSERT INTO my_duckdb_table
SELECT *
FROM postgres_scan_pushdown(
    '<connect string>, '<schema>', '<table>', min_ctid=280000
)
WHERE created_at > '<last_run_at>'
```

I could probably reduce the runtime by an additional 75% of an already super fast transfer.

Lmk what you think!

**Comment by Tishj** [comment]
Could you post this in the repository for the postgres_scanner?
Thanks 👍

**Reply by ben-pr-p** [reply]
Will do! Thanks!

**Reply by ben-pr-p** [reply]
Created the issue here: https://github.com/duckdb/postgres_scanner/issues/119
